contracts/VotingUtils.sol

```diff
@@ -379,4 +390,13 @@ contract VotingUtils is EternalStorage, VotingBase {
   function _setRejected(uint256 _id, uint256 _value) internal {
     uintStorage[keccak256(abi.encodePacked("votingState", _id, "rejected"))] = _value;
   }
+
+  function _checkTurnout(uint256 _id) internal view returns(bool) {
```

Comment:
(and so can be called `checkTurnout)

Reply:
Can make this public but not sure if it would be useful since it takes the amount staked at time of call and accepts and rejects only get update on cycleEnd